src/backend/bin/www

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env node
+/* eslint-disable no-restricted-globals */
+/* eslint-disable no-console */
+/* eslint-disable no-shadow */
+/* eslint-disable no-use-before-define */
+/* eslint-disable quotes */
+/**
+ * Module dependencies.
+ */
+
+const http = require("http");
+const debug = require("debug")("backend:server");
+const app = require("../app");
+
+
+/**
+ * Get port from environment and store in Express.
+ */
+
+const port = normalizePort(process.env.PORT || "5000");
+app.set("port", port);
+
+/**
+ * Create HTTP server.
+ */
+
+const server = http.createServer(app);
+
+/**
+ * Listen on provided port, on all network interfaces.
+ */
+
+server.listen(port);
+server.on("error", onError);
+server.on("listening", onListening);
+
+/**
+ * Normalize a port into a number, string, or false.
+ */
+
+function normalizePort(val) {
```

Comment:
Let us stick with es6 arrow functions for consistency

Reply:
okay